pkg/utils/utils.go

```diff
@@ -236,6 +236,11 @@ func (u *utils) IsBMInfra() (bool, error) {
 		return false, err
 	}
 
+	if infra.Status.PlatformStatus.Type == configv1.NonePlatformType {
+		log.Println("Detected platform type as 'None'; assuming cloud by default. " +
```

Comment:
This will be printed only if the logger is enabled?
Should this be printed as a warning?
Should utils print user messages? I think this print should be done in cmd